src/azure-cli/azure/cli/command_modules/appservice/_help.py

```diff
@@ -1668,6 +1668,52 @@
 long-summary: This command may not work with web apps running on Linux.
 """
 
+helps['webapp log deployment'] = """
+type: group
+short-summary: Manage web app deployment logs.
+"""
+
+helps['webapp log deployment show'] = """
```

Comment:
@btardif can you please review the help content & let us know if we need to update these? Thanks!